shared/locales/sv/translation.json

```
@@ -195,6 +195,11 @@
   ],
   "homeHeroGradientSubtext": "Från koncept till kod skapar vi människocentrerade GenAI-upplevelser.",
   "homeHeroSubtext": "Från koncept till kod skapar vi människocentrerade GenAI-upplevelser.",
+  "homeHeroSubtextOptions": [
+    "Från koncept till kod skapar vi människocentrerade GenAI-upplevelser.",
+    "Från koncept till kod, design och utveckling i ett team.",
```

Biome (2.1.2)

[error] The key homeHeroSubtext was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)